airflow/api_fastapi/common/db/common.py

```diff
@@ -52,23 +52,21 @@ def your_route(session: Annotated[Session, Depends(get_session)]):
         yield session
 
 
-def apply_filters_to_select(
-    *, base_select: Select, filters: Sequence[BaseParam | None] | None = None
-) -> Select:
+def apply_filters_to_select(*, query: Select, filters: Sequence[BaseParam | None] | None = None) -> Select:
```

Comment:
What do you think about `select_obj` since this is not yet a query? It becomes a query when we use `session.scalars`, `session.execute` etc.

Reply:
[query](https://docs.sqlalchemy.org/en/14/orm/query.html#query-api) is also a sqlalchemy function. But, I think, we can just leave it to be `select` cause, within the function, there's no need for sqlalchemy's select. If it's confusing, we can also make it `_select` or `select_obj`

Reply:
I vote for `select_obj` too. `select` and `query` are both in the SQLAchemy namespace. Also as Ephraim mentioned a `Select` is not the same as a `Query`.

+1 for removing the `base` in every name.

Reply:
yeah i'm aware of `Query` but i don't think there is a `query` free function. query is a common word for what you would pass here.

Reply:
query is a method on session; but this of course does not collide with that.

Reply:
See what i mean?

`query` is a colloquial term for "select statement"

Used in many places already
<img width="842" alt="image" src="https://github.com/user-attachments/assets/b07bd878-ae75-425b-9bae-befddfca7a95">

Reply:
also, @ephraimbuddy i'm not sure what you mean.  it beomes some kind of a "result" object after running through `scalars`.  

but, ok, if y'all hate `query`, we can do `statement`. let me know.

Reply:
Naming is hard. I agree `query` is bad choice. 

How about `select_statement` as a name ? Sounds pretty obvious to me - I think `select_obj` is proably worse idea  than `query`.

Reply:
but just in the spirit of trolling notice also
<img width="617" alt="image" src="https://github.com/user-attachments/assets/1047fbbc-8418-4c54-8525-d673bf969e51">

and 
<img width="605" alt="image" src="https://github.com/user-attachments/assets/17f1bf46-4418-472b-9337-81fc01932187">

another long-existing example noticed upon while updating to `statement`